dockerfiles/platform-build/Dockerfile

```diff
@@ -65,13 +65,28 @@ RUN mono `./vcpkg fetch nuget | tail -n 1` \
     -username "${GITHUB_ACTOR}" \
     -password "${GITHUB_TOKEN}"
 
+
 WORKDIR /hpcc-dev
 RUN mkdir build
 WORKDIR /hpcc-dev/build
 
+ARG SIGN_MODULES=OFF
+RUN --mount=type=secret,id=key,uid=10000 if [[ "${SIGN_MODULES}" != "OFF" ]] ; \
```

Comment:
The uid and gid are confirable (via helm values), so uid is not necessarily 10000.
I'm not sure of the sifnificance/nee for it to be 10000 here though?
Could/should this mount/command run as user root instead?

Reply:
The 10000 uid is hardcoded on line 33 for our docker builds.  I don't believe this 'hpcc' user is the same one that gets configured within our helm code. Mounting it as root causes issues because I want the key to be visible in the gpg keychain for the hpcc user for when we run our cmake commands.

Reply:
ok yes, this is the build image, the platform-core image uid and gid are configurable.